supplementary/alica_codegeneration/alica-plan-designer-fx-codegeneration/src/main/java/de/unikassel/vs/alica/generator/StandaloneCodegenerator.java

```diff
@@ -21,9 +21,10 @@ public class StandaloneCodegenerator {
     private static String tasksPath;
     private static String pluginsPath;
     private static String defaultPluginName = "DefaultPlugin";
+    private static String packageName = "lbc";
 
     private static void printUsage(){
-        System.out.println("Usage: java -jar StandaloneCodegenerator <clangFormat> <sourceGenPath> <plansPath> <rolesPath> <tasksPaths> <pluginsPath> [DefaultPluginName - default is \"DefaultPlugin\"]");
+        System.out.println("Usage: java -jar StandaloneCodegenerator <clangFormat> <sourceGenPath> <plansPath> <rolesPath> <tasksPaths> <pluginsPath> [DefaultPluginName - default is \"DefaultPlugin\"] [PackageName - default is \"lbc\"]");
```

Comment:
nit: Could consider just hardcoding the plugin to DefaultPlugin and removing as an arg.